py3status/modules/net_rate.py

```diff
@@ -5,41 +5,56 @@
 Configuration parameters:
     all_interfaces: ignore self.interfaces, but not self.interfaces_blacklist
         (default True)
-    cache_timeout: how often we refresh this module in seconds (default 2)
-    devfile: location of dev file under /proc (default '/proc/net/dev')
-    format: format of the module output (default '{interface}: {total}')
-    format_no_connection: when there is no data transmitted from the
-        start of the plugin (default '')
-    hide_if_zero: hide indicator if rate == 0 (default False)
-    interfaces: comma separated list of interfaces to track (default '')
+    cache_timeout: how often we refresh this module in seconds
+        (default 2)
+    devfile: location of dev file under /proc
+        (default '/proc/net/dev')
+    format: format of the module output
+        (default '{interface}: {total}')
+    format_no_connection: when there is no data transmitted from the start of the plugin
+        (default '')
+    format_value: format to use for values
+        (default "{value:5.1f} {unit:>5s}")
+    hide_if_zero: hide indicator if rate == 0
+        (default False)
+    interfaces: comma separated list of interfaces to track
+        (default [])
     interfaces_blacklist: comma separated list of interfaces to ignore
         (default 'lo')
-    precision: amount of numbers after dot (default 1)
+    si_units: use SI units
+        (default False)
```

Comment:
Of course, but I think it has its place. I'd rather we ask the user to be explicit than try and guess what they want.